pages/reference/base-images/base-images.md

```diff
@@ -108,92 +107,97 @@ CMD ["node", "main.js"]
 
 ### Supported Architectures, Distros and Languages
 
-Currently balenalib supports the following OS distributions and Language stacks, if you would like to see others added, create an issue on the [balena base images repo](https://github.com/balena-io-library/base-images/issues).
+Currently, balenalib supports the following OS distributions and Language stacks, if you would like to see others added, create an issue on the [balena base images repo]({{ $links.githubLibrary }}/base-images/issues).
+
+| Distribution | Default                      | Supported Architectures                      |
```

Comment:
Presumably in both this table and the next, the "Default" column is the version you'd get if you used the `latest` tag for the `<distro_ver>` or `<lang_ver>` fields?
Is it worth mentioning that explicitly? (e.g. by changing the "Default" column-header to "Default (latest)" ? ) :man_shrugging: 

---

Hmmm, in the "Multiple Distributions:" and "Multiple language stacks:" lists earlier on in the page, I wonder if it'd be worth highlighting the versions corresponding to the "latest" tag by e.g. displaying them in bold? Like:
 - [Node.js](https://nodejs.org/en/): 8.11.1, 8.17.0, 9.11.2, 10.18.0, 11.15.0, 12.14.0 and **13.5.0**
 - [Python](https://www.python.org/): **2.7.17**, 3.4.10, 3.5.7, 3.6.9, 3.7.5 and 3.8.0